ci/flip-production-traffic.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -u
+
+function log {
+   echo "$(date +"%T") - INFO - $*"
+}
+
+function read_version () {
+    CLUSTER=$1
+    log "Reading ${CLUSTER} version"
+    VERSION=$(gcloud app versions list --project="${CLUSTER}" --hide-no-traffic --service=default | grep "default" | tr -s " " | cut -f 2  -d\ )
```

Comment:
I've seen those pipe _tricks_ elsewhere. If you want to continue working with _text_:
```bash
gcloud app versions list --project=akvoflow-uat1 --hide-no-traffic --service=default | awk '/^default/ {print $2}'
```
Or `gcloud` has `--format=json` for structured output
```bash
gcloud app versions list --project=akvoflow-uat1 --hide-no-traffic --service=default --format=json | jq -M -r '.[].version.id'
```

Reply:
opted for --format="value(VERSION.id)"